spdx/package.py

```diff
@@ -127,6 +129,18 @@ def validate(self, messages=None):
 
         return messages
 
+    def validate_files_analyzed(self, messages):
+        if self.files_analyzed not in [ True, False, None ]:
+            messages = messages + [
+                'Package files_analyzed must be True or False or None (omitted)'
+            ]
+        if self.files_analyzed == False and self.verif_code is not None:
```

Comment:
```suggestion
        if self.files_analyzed is False and self.verif_code:
```

Reply:
```suggestion
        if self.files_analyzed is False and self.verif_code is not None:
```

I would keep the check that verif_code "is not None": if someone, by mistake, puts an empty string instead of omitting the tag, I think that it would better *not* to validate the file. What do you think?

Reply:
The pythonic way would be IMHO:
`if not self.files_analyzed is False and self.verif_code:`